packages/manager/apps/telecom/src/app/telecom/telephony/alias/portability/portabilities/attach/portabilities-attach.html

```diff
@@ -0,0 +1,43 @@
+<oui-modal data-heading="{{:: 'portabilities_attach_title' | translate }}"
+    data-primary-action="$ctrl.uploadFile()"
+    data-primary-label= "{{:: 'submit' | translate }}"
```

Comment:
I think you should scope your translations 
```suggestion
    data-primary-label= "{{:: 'telephony_alias_portability_attach_submit' | translate }}"
```